MELA/src/TUtil.cc

```diff
@@ -1522,9 +1528,65 @@ void TUtil::SetCKMElements(double* invckm_ud, double* invckm_us, double* invckm_
   cabib_.Vcb = __modparameters_MOD_ckmbare(&i, &j);
   // Do not call ckmfill_(), it is called by MCFM_chooser!
 }
+void TUtil::SetMadgraphCKMElements(double ckmlambda, double ckma, double ckmrho, double ckmeta){
+  if(ckmlambda >= 0){
+    madMela::params_r_.mdl_ckmlambda = ckmlambda;
+  }
+  if(ckma >= 0){
+    madMela::params_r_.mdl_ckma = ckma;
+  }
+  if(ckmrho >= 0){
+    madMela::params_r_.mdl_ckmrho = ckmrho;
+  }
+  if(ckmeta >= 0){
+    madMela::params_r_.mdl_ckmeta = ckmeta;
+  }
+  madMela::update_all_coup_();
```

Comment:
Does this call really update all couplings? If so, isn't it better to move to a later stage when the ME is actually called (like calls to chooser in MCFM)?

Reply:
My thought process was that one could update the CKM matrix and its dependent calculations right after editing the elements, but this makes sense as well.